Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -60,7 +60,7 @@ FNetworkGUID USpatialPackageMapClient::ResolveEntityActor(AActor* Actor, Worker_
 	return NetGUID;
 }
 
-void USpatialPackageMapClient::RemoveEntityActor(const Worker_EntityId& EntityId)
```

Comment:
Was removed due to @davedissian comment about a non ref would be faster.

Reply:
Nit: remove the `const` as well.